kube-weave weave-npc failing to start

Hi there

I am launch a `kops` weave cluster with private networking, @bboreham knows what I mean.  We have 20 nodes, and on one of them `weave-npc` is failing to start.  The issue is that ulogd is failing with:

```console
 # ulogd -v
Thu Nov 17 21:21:30 2016 <5> ulogd.c:843 building new pluginstance stack: 'log1:NFLOG,base1:BASE,pcap1:PCAP'
Thu Nov 17 21:21:30 2016 <7> ulogd_inppkt_NFLOG.c:552 unable to bind to log group 86
Thu Nov 17 21:21:30 2016 <7> ulogd.c:813 error starting `log1'
Thu Nov 17 21:21:30 2016 <8> ulogd.c:1430 not even a single working plugin stack
```

I have launched a pod onto the impacted node with a nodeSelector and am running from a single weave-npc pod.  

Here is a full strace for ulogd

```console
/ # strace -s 1024 ulogd -v
execve("/usr/sbin/ulogd", ["ulogd", "-v"], [/* 26 vars */]) = 0
arch_prctl(ARCH_SET_FS, 0x7f65bd3adb28) = 0
set_tid_address(0x7f65bd3adb60)         = 25878
mprotect(0x7f65bd3aa000, 4096, PROT_READ) = 0
mprotect(0x55d722f3a000, 4096, PROT_READ) = 0
access("/etc/ulogd.conf", R_OK)         = 0
open("/etc/ulogd.conf", O_RDONLY)       = 3
readv(3, [{"", 0}, {"[global]\nlogfile=\"/dev/null\"\nplugin=\"/usr/lib/ulogd/ulogd_inppkt_NFLOG.so\"\nplugin=\"/usr/lib/ulogd/ulogd_raw2packet_BASE.so\"\nplugin=\"/usr/lib/ulogd/ulogd_output_PCAP.so\"\nstack=log1:NFLOG,base1:BASE,pcap1:PCAP\n\n[log1]\ngroup=86\n\n[pcap1]\nfile=\"/var/log/ulogd.pcap\"\nsync=1\n\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0", 1024}], 2) = 268
open("/dev/null", O_WRONLY|O_CREAT|O_APPEND, 0666) = 4
fcntl(4, F_GETFL)                       = 0x8401 (flags O_WRONLY|O_APPEND|O_LARGEFILE)
ioctl(4, TIOCGWINSZ, 0x7ffe7ee9f108)    = -1 ENOTTY (Not a tty)
open("/usr/lib/ulogd/ulogd_inppkt_NFLOG.so", O_RDONLY|O_CLOEXEC) = 5
fcntl(5, F_SETFD, FD_CLOEXEC)           = 0
fstat(5, {st_mode=S_IFREG|0755, st_size=18816, ...}) = 0
read(5, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0 \22\0\0\0\0\0\0@\0\0\0\0\0\0\0\300C\0\0\0\0\0\0\0\0\0\0@\0008\0\6\0@\0\27\0\26\0\1\0\0\0\5\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0|\"\0\0\0\0\0\0|\"\0\0\0\0\0\0\0\0 \0\0\0\0\0\1\0\0\0\6\0\0\0\230,\0\0\0\0\0\0\230, \0\0\0\0\0\230, \0\0\0\0\0`\26\0\0\0\0\0\0\270\26\0\0\0\0\0\0\0\0 \0\0\0\0\0\2\0\0\0\6\0\0\0\310,\0\0\0\0\0\0\310, \0\0\0\0\0\310, \0\0\0\0\0\220\1\0\0\0\0\0\0\220\1\0\0\0\0\0\0\10\0\0\0\0\0\0\0P\345td\4\0\0\0@ \0\0\0\0\0\0@ \0\0\0\0\0\0@ \0\0\0\0\0\0\\\0\0\0\0\0\0\0\\\0\0\0\0\0\0\0\4\0\0\0\0\0\0\0Q\345td\6\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\20\0\0\0\0\0\0\0R\345td\4\0\0\0\230,\0\0\0\0\0\0\230, \0\0\0\0\0\230, \0\0\0\0\0h\3\0\0\0\0\0\0h\3\0\0\0\0\0\0\1\0\0\0\0\0\0\0\3\0\0\0003\0\0\0\1\0\0\0\6\0\0\0\212\3000\1\0\6@\v3\0\0\0006\0\0\0009\0\0\0@:\353,BE\325\354\273\343\222|\270\215\361\0168\205\230|\331qX\34\353\323\357\16\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\3\0\5\0\210\20\0\0\0\0\0\0\0\0\0\0\0\0\0\0003\2\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\260\2\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0=\1\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\314\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\302\2\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\374\2\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\27\3\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0c\2\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\v\1\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\351\1\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\220\2\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0-\3\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\376\1\0\0\20\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\324\2\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0o\2\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0C\0\0\0\"\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\250\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0~\2\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0Q\2\0\0\20\0\0\0\0\0\0\0\0\0\0\0", 960) = 960
mmap(NULL, 2117632, PROT_READ|PROT_EXEC, MAP_PRIVATE, 5, 0) = 0x7f65bcf1e000
mmap(0x7f65bd120000, 12288, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED, 5, 0x2000) = 0x7f65bd120000
close(5)                                = 0
open("/etc/ld-musl-x86_64.path", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/lib/libnetfilter_log.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/usr/local/lib/libnetfilter_log.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/usr/lib/libnetfilter_log.so.1", O_RDONLY|O_CLOEXEC) = 5
fcntl(5, F_SETFD, FD_CLOEXEC)           = 0
fstat(5, {st_mode=S_IFREG|0755, st_size=13560, ...}) = 0
read(5, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0x\21\0\0\0\0\0\0@\0\0\0\0\0\0\0\2700\0\0\0\0\0\0\0\0\0\0@\0008\0\6\0@\0\21\0\20\0\1\0\0\0\5\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\320(\0\0\0\0\0\0\320(\0\0\0\0\0\0\0\0 \0\0\0\0\0\1\0\0\0\6\0\0\0\210-\0\0\0\0\0\0\210- \0\0\0\0\0\210- \0\0\0\0\0\220\2\0\0\0\0\0\0\230\2\0\0\0\0\0\0\0\0 \0\0\0\0\0\2\0\0\0\6\0\0\0\210-\0\0\0\0\0\0\210- \0\0\0\0\0\210- \0\0\0\0\0@\1\0\0\0\0\0\0@\1\0\0\0\0\0\0\10\0\0\0\0\0\0\0P\345td\4\0\0\0(\"\0\0\0\0\0\0(\"\0\0\0\0\0\0(\"\0\0\0\0\0\0<\1\0\0\0\0\0\0<\1\0\0\0\0\0\0\4\0\0\0\0\0\0\0Q\345td\6\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\20\0\0\0\0\0\0\0R\345td\4\0\0\0\210-\0\0\0\0\0\0\210- \0\0\0\0\0\210- \0\0\0\0\0x\2\0\0\0\0\0\0x\2\0\0\0\0\0\0\1\0\0\0\0\0\0\0%\0\0\0\32\0\0\0\4\0\0\0\10\0\0\0A\20Hk\326\200\4H(\10\0\10\230\22\5\t\1\"U\"\30\247=\f\0\0\22\5\200\4\222@\32\0\0\0\0\0\0\0\0\0\0\0\33\0\0\0\0\0\0\0\34\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\37\0\0\0 \0\0\0!\0\0\0#\0\0\0&\0\0\0*\0\0\0,\0\0\0\0\0\0\0\0\0\0\0-\0\0\0\0\0\0\0.\0\0\0001\0\0\0002\0\0\0004\0\0\0005\0\0\0\0\0\0\0\0\0\0\0006\0\0\0\0\0\0\0008\0\0\0:\0\0\0\0\0\0\0<\0\0\0=\0\0\0\0\0\0\0>\0\0\0\rg\261\v\353\232\0\f\222\211F\337.\276\365\25\23VU\220cgbh\261T\311H\"\35\252\2033;L\1\214\352\251\203\376\2645l\325w\206\251p\3433\343\346\221J\345xi\313\337\273R\4\323\220\226\5H\33&\252\203\r\241L_\245\250\326 \230RL\200l$p\356%\330\260\257\263\300\252\304\2645\274\244\257\331\30j\331qX\34\273\343\222|\6Y\313\347\263\300\26\346\256\243$qCE\325\354r\233\223\213\255\20\336\222C;\370qK\2332e\234\3517\240\1\36-H\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\3\0\7\0x\21\0\0\0\0\0\0\0\0\0\0\0\0\0\0\324\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0n\3\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\257\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\331\1\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\376\1\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0m\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\17\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0", 960) = 960
mmap(NULL, 2113536, PROT_READ|PROT_EXEC, MAP_PRIVATE, 5, 0) = 0x7f65bcd1a000
mmap(0x7f65bcf1c000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED, 5, 0x2000) = 0x7f65bcf1c000
close(5)                                = 0
open("/lib/libnfnetlink.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/usr/local/lib/libnfnetlink.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/usr/lib/libnfnetlink.so.0", O_RDONLY|O_CLOEXEC) = 5
fcntl(5, F_SETFD, FD_CLOEXEC)           = 0
fstat(5, {st_mode=S_IFREG|0755, st_size=25680, ...}) = 0
read(5, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\20\26\0\0\0\0\0\0@\0\0\0\0\0\0\0\220`\0\0\0\0\0\0\0\0\0\0@\0008\0\6\0@\0\17\0\16\0\1\0\0\0\5\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\330R\0\0\0\0\0\0\330R\0\0\0\0\0\0\0\0 \0\0\0\0\0\1\0\0\0\6\0\0\0P]\0\0\0\0\0\0P] \0\0\0\0\0P] \0\0\0\0\0\260\2\0\0\0\0\0\0\260\2\0\0\0\0\0\0\0\0 \0\0\0\0\0\2\0\0\0\6\0\0\0P]\0\0\0\0\0\0P] \0\0\0\0\0P] \0\0\0\0\0000\1\0\0\0\0\0\0000\1\0\0\0\0\0\0\10\0\0\0\0\0\0\0P\345td\4\0\0\0\220E\0\0\0\0\0\0\220E\0\0\0\0\0\0\220E\0\0\0\0\0\0\374\1\0\0\0\0\0\0\374\1\0\0\0\0\0\0\4\0\0\0\0\0\0\0Q\345td\6\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\20\0\0\0\0\0\0\0R\345td\4\0\0\0P]\0\0\0\0\0\0P] \0\0\0\0\0P] \0\0\0\0\0\260\2\0\0\0\0\0\0\260\2\0\0\0\0\0\0\1\0\0\0\0\0\0\0%\0\0\0\35\0\0\0\10\0\0\0\t\0\0\0\2\1\v\204\240\211\2 \20\31\240d\"\200\20\\\0\0\0\26\312\2\2\f\0\4@\f8\0 \0\310 \3\210\202D\2\20\232\4AA\25\t\340\30\f\0\1\200$\1B\211\24\7\5\211\f\1\0\1\35\0\0\0\36\0\0\0 \0\0\0\"\0\0\0\0\0\0\0\0\0\0\0$\0\0\0&\0\0\0)\0\0\0+\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0-\0\0\0\0\0\0\0\0\0\0\0.\0\0\0000\0\0\0001\0\0\0003\0\0\0005\0\0\0008\0\0\0009\0\0\0<\0\0\0?\0\0\0B\0\0\0F\0\0\0G\0\0\0J\0\0\0K\0\0\0N\0\0\0O\0\0\0R\0\0\0S\0\0\0U\0\0\0V\0\0\0_\210;8j\353\23A1\220\370Y<\r;8\327\3104\214\32N\2465\311\25YT\220E\307\222=\264\315V\350\t\177>\364\24y>\3436E\217\344J\353,{=#\177\236\7\36\253}J7\rA\241*#*\2\266\"\203\177:8\2738\256\25T\226\370YW\301\344\375\6\217\213\10\321E\27\377\20&=\216\2\33C\216\261v\24\321/\"W!LhE\27H\256\370m\37\2\306\354`\272\315V\266q\304\365\331qX\34$P\352{\272\343\222|\37\343\3412n\220\370Yv\321C-\246\302]@1J\23\322\251K\2263\3367?\22\242\262\236\231\303G68/\317S\314X\0179\307\310%\201?CE\325\354{\264\315V*\241\236\231J\353\343\303\33\243\320\367\3654;\361\246\264\10\4'C\275Z{|w\255\242\322\33\323d\t98wy\330\32\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\3\0\7\0\20\26\0\0\0\0\0\0\0\0\0\0\0\0\0\0\212\3\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\261\1\0\0\22\0\0\0\0\0\0\0\0\0\0\0", 960) = 960
mmap(NULL, 2121728, PROT_READ|PROT_EXEC, MAP_PRIVATE, 5, 0) = 0x7f65bcb14000
mmap(0x7f65bcd19000, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED, 5, 0x5000) = 0x7f65bcd19000
close(5)                                = 0
mprotect(0x7f65bd120000, 4096, PROT_READ) = 0
mprotect(0x7f65bcf1c000, 4096, PROT_READ) = 0
mprotect(0x7f65bcd19000, 4096, PROT_READ) = 0
open("/usr/lib/ulogd/ulogd_raw2packet_BASE.so", O_RDONLY|O_CLOEXEC) = 5
fcntl(5, F_SETFD, FD_CLOEXEC)           = 0
fstat(5, {st_mode=S_IFREG|0755, st_size=14808, ...}) = 0
read(5, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\370\6\0\0\0\0\0\0@\0\0\0\0\0\0\0\0304\0\0\0\0\0\0\0\0\0\0@\0008\0\6\0@\0\27\0\26\0\1\0\0\0\5\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\364\23\0\0\0\0\0\0\364\23\0\0\0\0\0\0\0\0 \0\0\0\0\0\1\0\0\0\6\0\0\0\370\35\0\0\0\0\0\0\370\35 \0\0\0\0\0\370\35 \0\0\0\0\0X\25\0\0\0\0\0\0\270\25\0\0\0\0\0\0\0\0 \0\0\0\0\0\2\0\0\0\6\0\0\0(\36\0\0\0\0\0\0(\36 \0\0\0\0\0(\36 \0\0\0\0\0p\1\0\0\0\0\0\0p\1\0\0\0\0\0\0\10\0\0\0\0\0\0\0P\345td\4\0\0\0\364\21\0\0\0\0\0\0\364\21\0\0\0\0\0\0\364\21\0\0\0\0\0\0L\0\0\0\0\0\0\0L\0\0\0\0\0\0\0\4\0\0\0\0\0\0\0Q\345td\6\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\20\0\0\0\0\0\0\0R\345td\4\0\0\0\370\35\0\0\0\0\0\0\370\35 \0\0\0\0\0\370\35 \0\0\0\0\0\10\2\0\0\0\0\0\0\10\2\0\0\0\0\0\0\1\0\0\0\0\0\0\0\3\0\0\0\f\0\0\0\1\0\0\0\6\0\0\0\210\3000\1\0\4@\v\f\0\0\0\16\0\0\0\21\0\0\0BE\325\354\273\343\222|\270\215\361\0168\205\230|\331qX\34\353\323\357\16\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\3\0\5\0\230\6\0\0\0\0\0\0\0\0\0\0\0\0\0\0\224\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0C\0\0\0\"\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\241\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0R\0\0\0 \0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0)\0\0\0 \0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\r\0\0\0 \0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\233\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\247\0\0\0\20\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\200\0\0\0 \0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0j\0\0\0 \0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\323\0\0\0\20\0\23\0P3 \0\0\0\0\0\0\0\0\0\0\0\0\0\346\0\0\0\20\0\24\0\2603 \0\0\0\0\0\0\0\0\0\0\0\0\0\1\0\0\0\22\0\5\0\230\6\0\0\0\0\0\0\1\0\0\0\0\0\0\0\2\0\0\0\22\0\10\0\370\6\0\0\0\0\0\0\f\0\0\0\0\0\0\0\332\0\0\0\20\0\24\0P3 \0\0\0\0\0\0\0\0\0\0\0\0\0\7\0\0\0\22\0\t\0\232\21\0\0\0\0\0\0\1\0\0\0\0\0\0\0\0_init\0_fini\0_ITM_deregisterTMCloneTable\0_ITM_registerTMCloneTab", 960) = 960
mmap(NULL, 2113536, PROT_READ|PROT_EXEC, MAP_PRIVATE, 5, 0) = 0x7f65bc910000
mmap(0x7f65bcb11000, 12288, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED, 5, 0x1000) = 0x7f65bcb11000
close(5)                                = 0
mprotect(0x7f65bcb11000, 4096, PROT_READ) = 0
open("/usr/lib/ulogd/ulogd_output_PCAP.so", O_RDONLY|O_CLOEXEC) = 5
fcntl(5, F_SETFD, FD_CLOEXEC)           = 0
fstat(5, {st_mode=S_IFREG|0755, st_size=11104, ...}) = 0
read(5, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0H\t\0\0\0\0\0\0@\0\0\0\0\0\0\0\240%\0\0\0\0\0\0\0\0\0\0@\0008\0\6\0@\0\27\0\26\0\1\0\0\0\5\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\f\20\0\0\0\0\0\0\f\20\0\0\0\0\0\0\0\0 \0\0\0\0\0\1\0\0\0\6\0\0\0\240\35\0\0\0\0\0\0\240\35 \0\0\0\0\0\240\35 \0\0\0\0\0008\7\0\0\0\0\0\0\220\7\0\0\0\0\0\0\0\0 \0\0\0\0\0\2\0\0\0\6\0\0\0\320\35\0\0\0\0\0\0\320\35 \0\0\0\0\0\320\35 \0\0\0\0\0\200\1\0\0\0\0\0\0\200\1\0\0\0\0\0\0\10\0\0\0\0\0\0\0P\345td\4\0\0\0\234\16\0\0\0\0\0\0\234\16\0\0\0\0\0\0\234\16\0\0\0\0\0\0L\0\0\0\0\0\0\0L\0\0\0\0\0\0\0\4\0\0\0\0\0\0\0Q\345td\6\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\20\0\0\0\0\0\0\0R\345td\4\0\0\0\240\35\0\0\0\0\0\0\240\35 \0\0\0\0\0\240\35 \0\0\0\0\0`\2\0\0\0\0\0\0`\2\0\0\0\0\0\0\1\0\0\0\0\0\0\0\3\0\0\0\26\0\0\0\1\0\0\0\6\0\0\0\210\3000\1\0\4@\v\26\0\0\0\30\0\0\0\33\0\0\0BE\325\354\273\343\222|\270\215\361\0168\205\230|\331qX\34\353\323\357\16\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\3\0\5\0\250\10\0\0\0\0\0\0\0\0\0\0\0\0\0\0\267\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0C\0\0\0\"\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\353\0\0\0\20\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\344\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\314\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\300\0\0\0\20\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0R\0\0\0 \0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0)\0\0\0 \0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0E\1\0\0\21\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\r\0\0\0 \0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\375\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\240\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\224\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\n\1\0\0\20\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0C\1\0\0\21\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\233\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\335\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\246\0\0\0\22\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\200\0\0\0 \0\0\0\0\0\0\0\0\0\0\0", 960) = 960
mmap(NULL, 2109440, PROT_READ|PROT_EXEC, MAP_PRIVATE, 5, 0) = 0x7f65bc70d000
mmap(0x7f65bc90e000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED, 5, 0x1000) = 0x7f65bc90e000
close(5)                                = 0
open("/lib/libpcap.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/usr/local/lib/libpcap.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/usr/lib/libpcap.so.1", O_RDONLY|O_CLOEXEC) = 5
fcntl(5, F_SETFD, FD_CLOEXEC)           = 0
fstat(5, {st_mode=S_IFREG|0755, st_size=235712, ...}) = 0
read(5, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0>\0\1\0\0\0\350x\0\0\0\0\0\0@\0\0\0\0\0\0\0\0\223\3\0\0\0\0\0\0\0\0\0@\0008\0\6\0@\0\27\0\26\0\1\0\0\0\5\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0lk\3\0\0\0\0\0lk\3\0\0\0\0\0\0\0 \0\0\0\0\0\1\0\0\0\6\0\0\0\330u\3\0\0\0\0\0\330u#\0\0\0\0\0\330u#\0\0\0\0\0b\34\0\0\0\0\0\0\250)\0\0\0\0\0\0\0\0 \0\0\0\0\0\2\0\0\0\6\0\0\0\240\206\3\0\0\0\0\0\240\206#\0\0\0\0\0\240\206#\0\0\0\0\0P\1\0\0\0\0\0\0P\1\0\0\0\0\0\0\10\0\0\0\0\0\0\0P\345td\4\0\0\0\370\35\3\0\0\0\0\0\370\35\3\0\0\0\0\0\370\35\3\0\0\0\0\0\224\v\0\0\0\0\0\0\224\v\0\0\0\0\0\0\4\0\0\0\0\0\0\0Q\345td\6\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\20\0\0\0\0\0\0\0R\345td\4\0\0\0\330u\3\0\0\0\0\0\330u#\0\0\0\0\0\330u#\0\0\0\0\0(\32\0\0\0\0\0\0(\32\0\0\0\0\0\0\1\0\0\0\0\0\0\0\305\0\0\0c\0\0\0 \0\0\0\v\0\0\0\242\n\0\300I\10\202\tA\26\204\24\202\215 \0\4\0\0\0 \220\10\24\260\0|\2\2\tp\4\31\0\0\0\0\30\2\0\302\34\30\301\204\10\322\24\0\0\0\0\10@\0\0\0\300\f\1@\10! \4( \201C\210\10\0\10b\0@P\201&D d\21\0%\225\r\0\30\240\10\200\1B(\10\f \202\0\0\0\0\21P\1\4DHHHC8\206!\20\1(\207X\6\0\302 \200\4\30\26\200\0\271\0`@!\234\222\1\311\220M\0\241v\0060D\201\0\21 \341`\6\2\4\7\t\1\220\0a\0$\20\0210@\10 \220$\244Q\10\250\0\10\20\0\0\0\2\0010\2\2244\4\4 \0\250\4\nH\201\203\0\0\300\nB\5d\20\210\ft0\0\0\2\350 \0\24D\0\10\10\20\4DIQ \344\0 \204\35\30\4\2AA`\207\20\7\10\2`$\20\2\3\3\4\0\1\211\2402$c\0\0\0\0\0\0\0e\0\0\0\0\0\0\0g\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0h\0\0\0k\0\0\0m\0\0\0p\0\0\0q\0\0\0r\0\0\0t\0\0\0w\0\0\0x\0\0\0|\0\0\0~\0\0\0\0\0\0\0\177\0\0\0\200\0\0\0\202\0\0\0\204\0\0\0\205\0\0\0\0\0\0\0\210\0\0\0\211\0\0\0\213\0\0\0\0\0\0\0\214\0\0\0\0\0\0\0\216\0\0\0\0\0\0\0\220\0\0\0\0\0\0\0\224\0\0\0\226\0\0\0\230\0\0\0\231\0\0\0\235\0\0\0\0\0\0\0\237\0\0\0\0\0\0\0\241\0\0\0\242\0\0\0\245\0\0\0\246\0\0\0\247\0\0\0\250\0\0\0\251\0\0\0\253\0\0\0\254\0\0\0\256\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\260\0\0\0\262\0\0\0\264\0\0\0\265\0\0\0\270\0\0\0\271\0\0\0\272\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\273\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\276\0\0\0", 960) = 960
mmap(NULL, 2334720, PROT_READ|PROT_EXEC, MAP_PRIVATE, 5, 0) = 0x7f65bc4d3000
mmap(0x7f65bc70a000, 12288, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED, 5, 0x37000) = 0x7f65bc70a000
close(5)                                = 0
mprotect(0x7f65bc90e000, 4096, PROT_READ) = 0
mprotect(0x7f65bc70a000, 8192, PROT_READ) = 0
clock_gettime(CLOCK_REALTIME, {1479417537, 496274100}) = 0
open("/etc/localtime", O_RDONLY|O_NONBLOCK|O_CLOEXEC) = 5
fstat(5, {st_mode=S_IFREG|0644, st_size=118, ...}) = 0
mmap(NULL, 118, PROT_READ, MAP_SHARED, 5, 0) = 0x7f65bd3a9000
close(5)                                = 0
writev(2, [{"Thu Nov 17 21:18:57 2016 <5> ulogd.c:843 ", 41}, {NULL, 0}], 2Thu Nov 17 21:18:57 2016 <5> ulogd.c:843 ) = 41
writev(4, [{"Thu Nov 17 21:18:57 2016 <5> ulogd.c:843 building new pluginstance stack: 'log1:NFLOG,base1:BASE,pcap1:PCAP'\n", 109}, {NULL, 0}], 2) = 109
writev(2, [{"building new pluginstance stack: 'log1:NFLOG,base1:BASE,pcap1:PCAP'\n", 68}, {NULL, 0}], 2building new pluginstance stack: 'log1:NFLOG,base1:BASE,pcap1:PCAP'
) = 68
brk(NULL)                               = 0x55d724d09000
brk(0x55d724d0b000)                     = 0x55d724d0b000
brk(0x55d724d0d000)                     = 0x55d724d0d000
open("/etc/ulogd.conf", O_RDONLY)       = 5
readv(5, [{"", 0}, {"[global]\nlogfile=\"/dev/null\"\nplugin=\"/usr/lib/ulogd/ulogd_inppkt_NFLOG.so\"\nplugin=\"/usr/lib/ulogd/ulogd_raw2packet_BASE.so\"\nplugin=\"/usr/lib/ulogd/ulogd_output_PCAP.so\"\nstack=log1:NFLOG,base1:BASE,pcap1:PCAP\n\n[log1]\ngroup=86\n\n[pcap1]\nfile=\"/var/log/ulogd.pcap\"\nsync=1\n\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0", 1024}], 2) = 268
readv(5, [{"", 0}, {"[global]\nlogfile=\"/dev/null\"\nplugin=\"/usr/lib/ulogd/ulogd_inppkt_NFLOG.so\"\nplugin=\"/usr/lib/ulogd/ulogd_raw2packet_BASE.so\"\nplugin=\"/usr/lib/ulogd/ulogd_output_PCAP.so\"\nstack=log1:NFLOG,base1:BASE,pcap1:PCAP\n\n[log1]\ngroup=86\n\n[pcap1]\nfile=\"/var/log/ulogd.pcap\"\nsync=1\n\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0", 1024}], 2) = 0
close(5)                                = 0
open("/etc/ulogd.conf", O_RDONLY)       = 5
readv(5, [{"", 0}, {"[global]\nlogfile=\"/dev/null\"\nplugin=\"/usr/lib/ulogd/ulogd_inppkt_NFLOG.so\"\nplugin=\"/usr/lib/ulogd/ulogd_raw2packet_BASE.so\"\nplugin=\"/usr/lib/ulogd/ulogd_output_PCAP.so\"\nstack=log1:NFLOG,base1:BASE,pcap1:PCAP\n\n[log1]\ngroup=86\n\n[pcap1]\nfile=\"/var/log/ulogd.pcap\"\nsync=1\n\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0", 1024}], 2) = 268
lseek(5, -34, SEEK_CUR)                 = 234
close(5)                                = 0
brk(0x55d724d32000)                     = 0x55d724d32000
socket(PF_NETLINK, SOCK_RAW, NETLINK_NETFILTER) = 5
getsockname(5, {sa_family=AF_NETLINK, pid=0, groups=00000000}, [12]) = 0
clock_gettime(CLOCK_REALTIME, {1479417537, 498032632}) = 0
bind(5, {sa_family=AF_NETLINK, pid=0, groups=00000000}, 12) = 0
getsockname(5, {sa_family=AF_NETLINK, pid=25878, groups=00000000}, [12]) = 0
bind(5, {sa_family=AF_NETLINK, pid=25878, groups=00000000}, 12) = 0
sendto(5, "\34\0\0\0\1\4\5\0\0\0\0\0\0\0\0\0\0\0\0V\5\0\1\0\1\0\0\0", 28, 0, {sa_family=AF_NETLINK, pid=0, groups=00000000}, 12) = 28
recvfrom(5, "0\0\0\0\2\0\0\0\0\0\0\0\26e\0\0\377\377\377\377\34\0\0\0\1\4\5\0\0\0\0\0\0\0\0\0\0\0\0V\5\0\1\0\1\0\0\0", 8192, 0, {sa_family=AF_NETLINK, pid=0, groups=00000000}, [12]) = 48
clock_gettime(CLOCK_REALTIME, {1479417537, 498577692}) = 0
writev(2, [{"Thu Nov 17 21:18:57 2016 <7> ulogd_inppkt_NFLOG.c:552 ", 54}, {NULL, 0}], 2Thu Nov 17 21:18:57 2016 <7> ulogd_inppkt_NFLOG.c:552 ) = 54
writev(4, [{"Thu Nov 17 21:18:57 2016 <7> ulogd_inppkt_NFLOG.c:552 unable to bind to log group 86\n", 85}, {NULL, 0}], 2) = 85
writev(2, [{"unable to bind to log group 86\n", 31}, {NULL, 0}], 2unable to bind to log group 86
) = 31
close(5)                                = 0
clock_gettime(CLOCK_REALTIME, {1479417537, 499088549}) = 0
writev(2, [{"Thu Nov 17 21:18:57 2016 <7> ulogd.c:813 ", 41}, {NULL, 0}], 2Thu Nov 17 21:18:57 2016 <7> ulogd.c:813 ) = 41
writev(4, [{"Thu Nov 17 21:18:57 2016 <7> ulogd.c:813 error starting `log1'\n", 63}, {NULL, 0}], 2) = 63
writev(2, [{"error starting `log1'\n", 22}, {NULL, 0}], 2error starting `log1'
) = 22
lseek(3, -52, SEEK_CUR)                 = 216
close(3)                                = 0
clock_gettime(CLOCK_REALTIME, {1479417537, 499672832}) = 0
writev(2, [{"Thu Nov 17 21:18:57 2016 <8> ulogd.c:1430 ", 42}, {NULL, 0}], 2Thu Nov 17 21:18:57 2016 <8> ulogd.c:1430 ) = 42
writev(4, [{"Thu Nov 17 21:18:57 2016 <8> ulogd.c:1430 not even a single working plugin stack\n", 81}, {NULL, 0}], 2) = 81
writev(2, [{"not even a single working plugin stack\n", 39}, {NULL, 0}], 2not even a single working plugin stack
) = 39
writev(2, [{"", 0}, {"Fatal error.\n", 13}], 2Fatal error.
) = 13
exit_group(1)                           = ?
+++ exited with 1 +++
/ # ulogd -v
Thu Nov 17 21:21:30 2016 <5> ulogd.c:843 building new pluginstance stack: 'log1:NFLOG,base1:BASE,pcap1:PCAP'
Thu Nov 17 21:21:30 2016 <7> ulogd_inppkt_NFLOG.c:552 unable to bind to log group 86
Thu Nov 17 21:21:30 2016 <7> ulogd.c:813 error starting `log1'
Thu Nov 17 21:21:30 2016 <8> ulogd.c:1430 not even a single working plugin stack
Fatal error.```

@chrislovecnm two possibilities come to mind so far, most likely first:

1) Something is already bound to that group - most likely another instance of ulogd - observe:

```
vagrant@vagrant-ubuntu-wily-64:~$ sudo docker run -tid --entrypoint=ulogd --privileged --net=host weaveworks/weave-npc:1.8.0 -v
87f7c2a86d7fb275b74c2ed5e1779b7453dd3234e47ef50b921ed0eabfef9f3f
vagrant@vagrant-ubuntu-wily-64:~$ docker logs 87f7c2a86d7fb275b74c2ed5e1779b7453dd3234e47ef50b921ed0eabfef9f3f
Fri Nov 18 12:32:31 2016 <5> ulogd.c:843 building new pluginstance stack: 'log1:NFLOG,base1:BASE,pcap1:PCAP'
vagrant@vagrant-ubuntu-wily-64:~$ sudo docker run -ti --entrypoint=ulogd --privileged --net=host weaveworks/weave-npc:1.8.0 -v
Fri Nov 18 12:32:50 2016 <5> ulogd.c:843 building new pluginstance stack: 'log1:NFLOG,base1:BASE,pcap1:PCAP'
Fri Nov 18 12:32:50 2016 <7> ulogd_inppkt_NFLOG.c:552 unable to bind to log group 86
Fri Nov 18 12:32:50 2016 <7> ulogd.c:813 error starting `log1'
Fri Nov 18 12:32:50 2016 <8> ulogd.c:1430 not even a single working plugin stack
Fatal error.
```

I wonder if perhaps we have somehow ended up with an orphaned ulogd process lying around on that machine?

2) The netlink bind [attempts to load some kernel modules](https://github.com/torvalds/linux/blob/v4.8/net/netfilter/nfnetlink_log.c#L174) - do you have `nfnetlink` and `nfnetlink_log` modules available on that machine?

```
vagrant@vagrant-ubuntu-wily-64:~$ lsmod|grep nfnetlink
vagrant@vagrant-ubuntu-wily-64:~$ find /lib/modules|grep nfnetlink
/lib/modules/4.2.0-27-generic/kernel/net/netfilter/nfnetlink_queue.ko
/lib/modules/4.2.0-27-generic/kernel/net/netfilter/nfnetlink_log.ko
/lib/modules/4.2.0-27-generic/kernel/net/netfilter/nfnetlink_acct.ko
/lib/modules/4.2.0-27-generic/kernel/net/netfilter/nfnetlink.ko
/lib/modules/4.2.0-27-generic/kernel/net/netfilter/nfnetlink_cttimeout.ko
/lib/modules/4.2.0-27-generic/kernel/net/netfilter/nfnetlink_cthelper.ko
vagrant@vagrant-ubuntu-wily-64:~$ sudo mv /lib/modules/4.2.0-27-generic/kernel/net/netfilter/nfnetlink_log.ko /tmp
vagrant@vagrant-ubuntu-wily-64:~$ sudo docker run -ti --entrypoint=ulogd --privileged --net=host weaveworks/weave-npc:1.8.0 -v
Fri Nov 18 12:18:37 2016 <5> ulogd.c:843 building new pluginstance stack: 'log1:NFLOG,base1:BASE,pcap1:PCAP'
Fri Nov 18 12:18:37 2016 <7> ulogd_inppkt_NFLOG.c:552 unable to bind to log group 86
Fri Nov 18 12:18:37 2016 <7> ulogd.c:813 error starting `log1'
Fri Nov 18 12:18:37 2016 <8> ulogd.c:1430 not even a single working plugin stack
Fatal error.
```

--- a/npc/ulogd/ulogd.go
+++ b/npc/ulogd/ulogd.go
@@ -4,6 +4,7 @@ import (
 	"io"
 	"os"
 	"os/exec"
+	"syscall"
 
 	"github.com/weaveworks/weave/common"
 )
@@ -17,6 +18,9 @@ func waitForExit(cmd *exec.Cmd) {
 
 func Start() error {
 	cmd := exec.Command("/usr/sbin/ulogd", "-v")
+	cmd.SysProcAttr = &syscall.SysProcAttr{
+		Pdeathsig: syscall.SIGKILL,
+	}
 	stdout, err := cmd.StderrPipe()
 	if err != nil {
 		return err

